sleap_nn/config/model_config.py

```
@@ -0,0 +1,9 @@
+@attr.s(auto_attribs=True)
```

Ruff

Undefined name `attr`

(F821)